### PR TITLE
Use get over keys

### PR DESCRIPTION
**Pull request by benrudolph**
@snopoke this should handle case attachments. don't know what the obsession was with getting all the keys for each method.

cc @orangejenny

### CODE REVIEW

**Comment by benrudolph** [comment]
(wait for tests)

File: dimagi/utils/django/cached_object.py [hunk 1]
```diff
@@ -259,28 +259,7 @@ def cache_put(self, image_stream, metadata):
         rcache.set(self.meta_key(OBJECT_ORIGINAL), simplejson.dumps(image_meta.to_json()))
 
     def is_cached(self):
-        try:
-            metas, streams = self.get_all_keys()
```
Comment (benrudolph) [inline dimagi/utils/django/cached_object.py]: this only checked for meta anyways and when the cache is put, it is always put as `OBJECT_ORIGINAL` so checking for that key is equivalent

**Comment by snopoke** [comment]
:+1:

FYI not mergable

**Comment by benrudolph** [comment]
did a whole bunch of work to upgrade redis so i could use `.exists` but we still can't use it haha. going with the `.get is not None` approach

**Comment by benrudolph** [comment]
k tested this out locally and is g2g

**Comment by orangejenny** [comment]
(leaving for @snopoke to merge)

**Comment by benrudolph** [comment]
@snopoke ping, if you get a chance to look at this today

**Pull request merged**